Dockerfile

```diff
@@ -47,5 +30,5 @@ RUN echo "Timestamp:" `date --utc` | tee /image-build-info.txt
 
 ENV X509_USER_PROXY /etc/grid-security/x509up
 
-CMD sh -c "python x509_updater.py --voms $VOMS"
+CMD sh -c "python3 x509_updater.py --voms $VOMS"
```

Comment:
This `CMD` is ignored in the helm chart and is set in the x509 deployment yaml. This PR will depend on a change in that chart